torch/fx/_symbolic_trace.py

```diff
@@ -250,6 +249,13 @@ def __init__(
         self.param_shapes_constant = param_shapes_constant
 
         self.submodule_paths: Optional[Dict[torch.nn.Module, str]] = None
+        self.root_module_name: str = ""
+        # Maps the containing module's name to the operator name
+        self.scope = Scope("", None)
+        # Records the module call stack
+        self.module_stack = {}
```

Comment:
should this be an OrderedDict?

Reply:
also nit: maybe rename this to `nn_module_stack` to make it clearer

Reply:
Added typing.OrderedDict to initialization

Done